malaria-ios/SocialShareAbleTest.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SocialShareAbleTest.swift
+//  malaria-ios
+//
+//  Created by Eric Huang on 1/7/16.
+//  Copyright © 2016 Bruno Henriques. All rights reserved.
+//
+
+import XCTest
+@testable import malaria_ios
+
+class SocialShareAbleTest: XCTestCase {
+    
+    var viewController: SocialMediaController!
+
+    override func setUp() {
+        super.setUp()
+        
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
```

Comment:
remove setup and tearDown method. They are just calling the super so they are doing nothing.